Remotion/Web/UnitTests/Core/UI/Controls/TabbedMenuImplementation/Rendering/TabbedMenuRendererTest.cs

```diff
@@ -106,6 +106,18 @@ public void RenderDiagnosticMetadataAttributes ()
       table.AssertAttributeValueEquals(DiagnosticMetadataAttributes.ControlType, "TabbedMenu");
     }
 
+    [Test]
+    public void RenderWebStrings ()
+    {
+      _control.Setup(_ => _.StatusText).Returns(WebString.CreateFromText("Multiline\nStatusText"));
+
+      _renderer.Render(new TabbedMenuRenderingContext(_httpContext.Object, _htmlHelper.Writer, _control.Object));
+
+      var document = _htmlHelper.GetResultDocument();
+      var node = document.SelectSingleNode("//*[@class=\"tabbedMenuStatusCell\"]");
+      Assert.That(node.InnerXml, Is.EqualTo("Multiline<br />StatusText"));
```

Comment:
two helper methods, GetAssertElementById and GetAssertedContentByClass would be great for this sort of thing.

var node = documentGetAssertedElementById(id);
Assert.That (node.InnerXml, Is.EqualTo(...);

This way, it would also make it easier to inject existing classes and IDs like it's done with the other tests.